packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginsStates.tsx

```diff
@@ -291,6 +287,11 @@ export default function(props: Props) {
 					onPluginSettingsChange={onSearchPluginSettingsChange}
 					renderDescription={props.renderDescription}
 					repoApi={repoApi}
+					setShouldRenderUserPlugins={setShouldRenderUserPlugins}
```

Comment:
Don't pass state handlers like this, that introduces tight coupling between component. Instead have the child component emit some event, and have the parent component react to this event (by calling setShouldRenderUserPlugins)